### PR TITLE
Fix default slippage on mainnet and not being persisted

### PR DESCRIPTION
**Pull request by walmat**
Fixes APP-1572

### CODE REVIEW

File: src/state/swaps/swapsStore.ts [hunk 1]
```diff
@@ -43,14 +43,14 @@ export const swapsStore = createRainbowStore<SwapsState>(
...
-    slippage: getDefaultSlippage(ChainId.mainnet, DEFAULT_CONFIG),
+    slippage: getDefaultSlippage(ChainId.mainnet, getRemoteConfig()),
     setSlippage: (slippage: string) => set({ slippage }),
     source: 'auto',
     setSource: (source: Source | 'auto') => set({ source }),
   }),
   {
     storageKey: 'swapsStore',
-    version: 1,
+    version: 2,
```
Comment (walmat) [inline src/state/swaps/swapsStore.ts:53]: not sure if we need a version bump when making changes to the store?

Comment (greg-schrammel) [reply src/state/swaps/swapsStore.ts:53]: only if you want to override the persisted config, in this case if you keep `version: 1` users that have already booted the app will keep the old default slippage

when do you want to get from remote config? rn it's gonna grab when the store is first created in the device and then persisted and never grab from `getRemoteConfig` again, as it's persisted

**Review by greg-schrammel — APPROVED** [review]

File: src/__swaps__/utils/swaps.ts [hunk 2]
```diff
@@ -380,11 +380,11 @@ export const DEFAULT_SLIPPAGE_BIPS = {
   [ChainId.blast]: 200,
 };
 
-export const slippageInBipsToString = (slippageInBips: number) => (slippageInBips / 100).toString();
+export const slippageInBipsToString = (slippageInBips: number) => (slippageInBips / 100).toFixed(1);
```
Comment (christianbaroni) [inline src/__swaps__/utils/swaps.ts:383]: Should this use safe math?

File: src/__swaps__/utils/swaps.ts [hunk 3]
```diff
@@ -380,11 +380,11 @@ export const DEFAULT_SLIPPAGE_BIPS = {
...
 
 export const slippageInBipsToStringWorklet = (slippageInBips: number) => {
   'worklet';
-  return (slippageInBips / 100).toFixed(1).toString();
+  return (slippageInBips / 100).toFixed(1);
```
Comment (christianbaroni) [inline src/__swaps__/utils/swaps.ts:387]: Same here

File: src/state/swaps/swapsStore.ts (hunk 1 above)
Comment (christianbaroni) [reply src/state/swaps/swapsStore.ts:53]: Maybe simpler to not persist default slippage?

Comment (christianbaroni) [reply src/state/swaps/swapsStore.ts:53]: Actually I don't think it should be persisted in its current form — would only make sense to me to persist if it were an explicit "default slippage" setting.

File: src/__swaps__/utils/swaps.ts (hunk 2 above)
Comment (walmat) [reply src/__swaps__/utils/swaps.ts:383]: i don't think this will ever be able to overflow, so it's pretty safe to not.

**Review by christianbaroni — APPROVED** [review]

**Pull request merged**